.github/workflows/rainix.yaml

```
@@ -1,5 +1,9 @@
 name: Rainix CI
-on: [push]
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)